frontend/src/metabase/css/core/flex.module.css

```diff
@@ -45,9 +45,9 @@
 
 .shrinkBelowContentSize {
   /* W3C spec says:
-     * By default, flex items won’t shrink below their minimum content size (the length of the longest word or
-     * fixed-size element). To change this, set the min-width or min-height property.
-     */
+         * By default, flex items won’t shrink below their minimum content size (the length of the longest word or
```

Comment:
This looks like a format change. Could we revert this so the PR only contains files about color classes migration?

Reply:
ok I've reverted them